### DISCUSSION TITLE
shuffle icon

### DISCUSSION THREAD

**Discussion by shavin-peiries** [tailwindlabs/heroicons#370] [Ideas]
My version is kind of shit when you put it next to heroicons 

![image](https://user-images.githubusercontent.com/63689399/120241407-4f7b1180-c280-11eb-9e42-ebd5d7ac14ba.png)

**Comment by ndri** [comment]
I added `arrows-crossing` to [Sidekickicons](https://github.com/ndri/sidekickicons/), which are meant to be used as a companion to Heroicons. Browse them here: https://sidekickicons.com/?icon=arrows-crossing
